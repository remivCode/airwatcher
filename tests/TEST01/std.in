1
a
a
1
2
0
0