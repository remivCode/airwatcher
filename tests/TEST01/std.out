1;
2;
Error: The radius must be strictly positive and must not exceed 200;